src/datastore/postgres/PgDataStore.ts

```diff
@@ -728,7 +728,7 @@ export class PgDataStore implements DataStore, ProvisioningStore {
         let currentVersion = await this.getSchemaVersion();
         while (currentVersion < PgDataStore.LATEST_SCHEMA) {
             log.info(`Updating schema to v${currentVersion + 1}`);
-            // eslint-disable-next-line @typescript-eslint/no-var-requires
+            // eslint-disable-next-line @typescript-eslint/no-var-requires,@typescript-eslint/no-require-imports
             const runSchema = require(`./schema/v${currentVersion + 1}`).runSchema;
```

Comment:
For the sake of avoiding `require()`, consider using a dynamic `import` instead:
```suggestion
            const runSchema = (await import(`./schema/v${currentVersion + 1}`)).runSchema;
```